pkg/controller/datavolume-controller_test.go

```diff
@@ -532,7 +616,14 @@ var _ = Describe("Reconcile Datavolume status", func() {
 				AnnDefaultStorageClass: "true",
 			},
 			storagev1.VolumeBindingWaitForFirstConsumer)
-		reconciler = createDatavolumeReconciler(sc, newImportDataVolume("test-dv"))
+		volumeMode := corev1.PersistentVolumeFilesystem
+		importDataVolume := newImportDataVolumeWithPvc("test-dv", &corev1.PersistentVolumeClaimSpec{
+			AccessModes: []corev1.PersistentVolumeAccessMode{corev1.ReadWriteOnce},
+			VolumeMode:  &volumeMode,
+		})
+
+		reconciler = createDatavolumeReconciler(sc, importDataVolume)
+
```

Comment:
yes, at least `AccessModes` is required. Actually it was required previously and we did enforce it in the webhook. So this test was not using the correct data as input (but the cdi logic did not check anything, just copied `dv.spec.PVC` to `PVC.spec`).  
Now the webhook is removed, since DV can be created without `AccessModes`, but this requires `StorageProfiles` to be correctly configured, and that is where the test will fail.

Reply:
So it's relevant only for the test and there is no potential regression for users here, right?

Reply:
yes, test is corrected no regression here (remember there was the DV webhook)